tempesta_fw/sched.c

```diff
@@ -60,7 +60,7 @@ tfw_sched_get_vhost(TfwMsg *msg)
 	list_for_each_entry_rcu(sched, &sched_list, list) {
 		if (!sched->sched_vhost)
 			break;
-		if ((vhost = sched->sched_vhost(msg))) {
+		if ((vhost = sched->sched_vhost(msg, block))) {
```

Comment:
It was said in one of the last reviews by @krizhanovsky : http (vhost) scheduler isn't one of possible schedulers and it's API is far away of hash and rr schedulers. Foreach cycle must be replaced with direct http (vhost) scheduler call. This wasn't done in this PR, please add issue for it.

Reply:
Done.